### PR TITLE
[FIX] Apicurito step fix

### PR DESCRIPTION
**Pull request by Stefan365**
//test: `@api-provider-prevent-publishing-empty-integration-by-deleting-all`

### CODE REVIEW

**Comment by Stefan365** [comment]
retest this please

**Comment by Stefan365** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by Stefan365** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
API Provider Integration | Publishing an integration with all operations deleted should not be possible 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by Stefan365** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/java/io/syndesis/qe/steps/apiprovider/ApiProviderSteps.java [hunk 1]
```diff
@@ -133,6 +135,7 @@ public void createTheTODOIntegration(String name, String source, String path) {
         cs.clickOnLink("Save");
         new CreateIntegrationSteps().setIntegrationName(name);
         cs.clickOnButton("Save");
+        Selenide.sleep(2000);
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/apiprovider/ApiProviderSteps.java]: https://selenide.org/javadoc/4.10/com/codeborne/selenide/Selenide.html
Selenide.sleep is not recommended. use `TestUtils.sleepWithoutInterrupt` or something like that

File: ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/ApicurioSteps.java [hunk 2]
```diff
@@ -326,6 +329,14 @@ public void changeFrameTo(String frame) {
         } else if ("syndesis".equals(frame)) {
             switchTo().parentFrame();
         }
+        log.info("FRAME CHANGED TO *{}*,", checkFrame());
+    }
+
+    private String checkFrame() {
+        WebDriver driver = WebDriverRunner.getWebDriver();
+        JavascriptExecutor jsExecutor = (JavascriptExecutor) driver;
+        String currentFrame = (String) jsExecutor.executeScript("return self.name");
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/ApicurioSteps.java]: you can use `executeJavaScript()` method

**Comment by syndesisqe** [comment]
Checkstyle verification failed with:
```
ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/
  ApicurioSteps.java:22:8: Unused import - org.openqa.selenium.JavascriptExecutor. [UnusedImports]
ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/
  ApicurioSteps.java:23:8: Unused import - org.openqa.selenium.WebDriver. [UnusedImports]
ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/
  ApicurioSteps.java:29:8: Unused import - com.codeborne.selenide.WebDriverRunner. [UnusedImports]
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**